src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -102,6 +128,39 @@ private void applyChanges( final Persistence<SutState> persistence, final SuiteC
 		}
 	}
 
+	private void checkForWarningMessage( final ReviewResult changeSets ) {
+		for ( final SuiteChangeSet suiteChangeSet : changeSets.getSuiteChangeSets() ) {
+			for ( final TestChangeSet testChangeSet : suiteChangeSet.getTestChangeSets() ) {
+				final ActionChangeSet actionChangeSet = testChangeSet.getInitialStateChangeSet();
+				final AttributeChanges attributeChanges = actionChangeSet.getIdentAttributeChanges();
+				for ( final Set<AttributeDifference> attributeDifferences : attributeChanges.getChanges().values() ) {
+					for ( final AttributeDifference attributeDifference : attributeDifferences ) {
+						final ElementIdentificationWarning warning =
+								attributeDifference.getElementIdentificationWarning();
+						if ( warning != null ) {
+							logger.warn( getWarningMessage( attributeDifference, warning ) );
+						}
+					}
+				}
+			}
+		}
+	}
+
+	private String getWarningMessage( final AttributeDifference attributeDifference,
+			final ElementIdentificationWarning warning ) {
+		final String title = "*************** recheck warning ***************";
+		final String elementIdentifier = attributeDifference.getKey();
+		final String expectedValue = attributeDifference.getExpectedToString();
+		final String actualValue = attributeDifference.getActualToString();
+		final String elementIdentifierInfo =
+				"The HTML " + elementIdentifier + " attribute used for element identification changed from "
+						+ expectedValue + " to " + actualValue + ".";
+		final String info = "recheck identified the element based on the persisted Golden Master.";
+		final String onApplyChangesInfo = "If you apply these changes to the Golden Master, your test "
+				+ warning.getTestClassName() + " will break.";
+		return title + "\n" + elementIdentifierInfo + "\n" + info + "\n" + onApplyChangesInfo;
```

Comment:
You could generally think about using `String#format( String, Object... )` to increase readability.